consul-on-ubuntu/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Deploys a 3 node Consul Cluster",
+  "description": "This template deploys a 3 node Consul cluster and auto-joins the nodes via Atlas",
```

Comment:
Can you add a bit more to the description to describe what Consul and Atlas are?